testsuites/kubernetes_versions.robot

```diff
@@ -92,7 +92,7 @@ Verify --wait flag works as expected
 
     Kubectl.Pods with prefix are running    default    wait-flag-good-nginx-ext-    3
     Kubectl.Return code should be   0
-    Kubectl.Pods with prefix are running    default    wait-flag-good-nginx-fluentd-es-    1
+    Kubectl.Pods with prefix are running    default    wait-flag-good-nginx-fluentd-es-    6
```

Comment:
I would run our tests in a dedicated namespace, instead of chaning the number of pods

Reply:
+1

Reply:
these are coming from the `testdata/charts/nginx/templates/ds.yaml` daemon set. Our cluster runs on 6 nodes, so 6 pods are coming up. we need to check for the `daemon set` here instead of pods. looking into it.

Reply:
yes, I agree with running all the tests in a dedicated namespace. I believe these tests are targeted for a kind cluster and `default` namespace is hard coded in the testsuite